pkgs/by-name/rc/rcp/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv
+, buildPackages
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+}:
```

Comment:
One dependency per line. Just like pretty much all other derivation in `nixpkgs`.